editor/src/clj/editor/outline_view.clj

```diff
@@ -471,13 +471,15 @@
                        (proxy-super setGraphic nil)
                        (proxy-super setContextMenu nil)
                        (proxy-super setStyle nil))
-                     (let [{:keys [label icon link outline-error? outline-overridden? outline-reference? outline-show-link? parent-reference? child-error? child-overridden? scene-visibility]} item
+                     (let [{:keys [label icon link color outline-error? outline-overridden? outline-reference? outline-show-link? parent-reference? child-error? child-overridden? scene-visibility]} item
```

Comment:
We specify color in `tile-source` `outline/OutlineData` but never really use it. Also we don't specify color anywhere else in `outline/OutlineData` (at least I can't find it)